### ISSUE TITLE
Need to show descriptions of each plan 

### ISSUE THREAD

**Issue by jacobsfletch** [payloadcms/website#230]
On the `/configure` screen, there is currently no way for the user to differentiate the different plans. We need to render a "?" icon that opens a drawer or tooltip with more information. We _don't_ want to simply redirect the user to the cloud-pricing page, though.

<img width="976" alt="Screenshot 2023-08-18 at 10 11 36 AM" src="https://github.com/payloadcms/website/assets/15735305/8b4eb0cb-9acd-43c8-9f01-d5ec5c16c0a5">

**Comment by JessChowdhury** [comment]
Over in the `cloud-cms` I have opened a [PR here](https://github.com/payloadcms/cloud-cms/pull/93) to add a plan description field.

Thenon [this branch](https://github.com/payloadcms/website/tree/feat/adds-plan-preview) - I've injected a drawer toggler into each line of the `planSelector` and added a drawer that renders the plan name + description.

TODO:
- Add icon (we have 'info' as a placeholder for now)
- Any further styling updates

**Comment by tylandavis** [comment]
@JessChowdhury @jacobsfletch What if instead of a drawer for each plan, we add a button 'Compare Plans' that opens up a drawer that shows the comparison chart? Would offer much less clicking I think.

**Comment by tylandavis** [comment]
@jacobsfletch draft PR here: #245.

**Comment by jacobsfletch** [comment]
@tylandavis I think this is probably a nice-to-have so if you can't get it working, the main goal here is to simply display the information to the user. They can still select another plan after exiting the drawer. BUT that being said, this should be pretty straightforward to implement. The drawer state needs to be entirely controlled by the checkout page, so you pass the value into the drawer, but then also lift it's changes out of the drawer back into the checkout. Something like this:

```
const [state, setState] = useState();

<MyDrawer
  value={state}
  onChange={setState}
/>
```

**Comment by jacobsfletch** [comment]
Closed with #245
